modules/installation-and-upgrade/pages/container-management/updating-server-containers.adoc

```diff
@@ -1,26 +1,72 @@
 = {productname} Server Upgrade
 
+// FIXME: Shouldn't we better say:
+//        "... to upgrade the host operating system first."
```

Comment:
+1. I agree, we should recommend to update the host OS 1st that will result in update of our tooling as well.